.travis.yml

```diff
@@ -14,3 +23,6 @@ env:
 addons:
   code_climate:
     repo_token: 2175628fb41494d45b50b6938600a53c130e1f7ac3b81b072bb71e91073a5e4a
```

Comment:
yeah, that's only of use to someone who wants to get code analysis done on this repo - won't work for anything else...